owa/app/js/components/view_log_entries/viewLogEntries.jsx

```diff
@@ -0,0 +1,135 @@
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+
+import React from 'react';
+import ReactTable from 'react-table';
+import 'react-table/react-table.css';
+import {FaSearch} from 'react-icons/lib/fa';
+import {Input} from 'reactstrap'
+
+export default class LogEntries extends React.Component {
+constructor() {
+super();
+this.data = []
+this.state = {
+isHidden: true
+}
+
+}
+advancedSearchView() {
+this.setState({
+isHidden: !this.state.isHidden
+})
+}
+render() {
+return (
+
+<div className="logwrapper">
+<div className="search_label">
+<i className=""></i>
+<span>Search Log Entries</span>
+</div>
+<div className="logSearch">
+<div className="input-group">
+<input type="text" placeholder="Search" className="log_searchbox"/>
+<span className="input-group-btn">
+<button className="btn btn-secondary" type="button">< FaSearch/></button>
+</span>
+</div>
+<button
+className="searchlink"
+onClick={this
+.advancedSearchView
+.bind(this)}>More Search Options</button>
+
+</div>
+<div className="logs_table_area">
+<div >{!this.state.isHidden && <AdvancedSearch/>}</div>
+<div className="table-responsive logs_table">
+<ReactTable
+data={this.data}
+defaultPageSize={(this.data.length < 1)
+? 5
+: (1 < this.data.length < 10)
+? 10
+: this.data.length}
+noDataText="No matching records found"
+filterAll={true}
+columns={[
+{
+accessor: 'source_name',
+Header: 'Source Name'
+}, {
+accessor: 'identifier',
+Header: 'Identifier'
+}, {
+accessor: 'date_gen',
+Header: 'Date Generated'
+}, {
+accessor: 'generated_by',
+Header: 'Generated By'
+}, {
+accessor: 'comments',
+Header: 'Comments'
+}
+]}/>
+</div>
+</div>
+</div>
+
+)
+}
+}
+
+const AdvancedSearch = () => (
+<div className="advanced_wrapper">
+<form>
+
+<br/>
+<fieldset>
+<div className="col-sm-6 col-md-4">
+<label className="search_lbl" name="source">Source Name</label>
+
+<Input type="select" id="source">
+<option>1</option>
+</Input>
+
+</div>
+<div className="col-sm-6 col-md-4">
+<label className="search_lbl" name="identifier">Identifier Contains</label>
+<Input id="identifier"/>
+</div>
+<div className="col-sm-6 col-md-4">
+<label className="search_lbl" name="gen_range">Generate Between</label>
+<Input id="date_lower"/>
+<span>-</span>
+<Input id="date_upper"/>
+</div>
+<div className="col-sm-6 col-md-4">
+<label className="search_lbl" name="gen_by">Generated By</label>
+<Input id="gen_by"/>
+</div>
+
+<div className="col-sm-6 col-md-4">
+<label className="search_lbl" name="comment">Comment Contains</label>
+<Input id="comment"/>
+</div>
+<div className="clear"></div>
+
+</fieldset>
+
+<div className="btn-group" role="group">
+<input type="submit" value="Search" className=" col-sm-6"/>
+<button className="col-sm-6">Clear All
+</button>
+
+</div>
+</form>
+</div>
+)
```

Comment:
Your indentation of this whole file went wild like real flat :-) On a less serious atmosphere, please fix this whole file's indentation.
Also please add a new line to the end of the file

Reply:
Haha, have no idea how that happened! Probably when I was clearing the double spaces and forgot to reformat. Sorted it now